cmd/cluster/dynatrace/cluster.go

```diff
@@ -148,3 +153,36 @@ func GetDynatraceURLFromManagementCluster(clusterID string) (string, error) {
 	DTURL := fmt.Sprintf("%s://%s", DTApiURL.Scheme, DTApiURL.Host)
 	return DTURL, nil
 }
+
+func getConnection() (conn *sdk.Connection, error error) {
+	connection, err := utils.CreateConnection()
+	if err != nil {
+		return nil, err
+	}
+	defer connection.Close()
+
+	return connection, nil
+}
+
+func getClientsetFromClusterID(clusterID string) (cset *kubernetes.Clientset, error error) {
+	_, _, clientset, err := common.GetKubeConfigAndClient(clusterID, "", "")
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve Kubernetes configuration and client for cluster with ID %s: %w", clusterID, err)
+	}
+
+	return clientset, nil
+}
```

Comment:
**nitpick:** this seems like a bit of a bloating wrapper.  It's just a function calling `common.GetKubeConfigAndClient` without any further logic. 

Why not directly have the following in the main logic?
```
	_, _, clientset, err := common.GetKubeConfigAndClient(clusterID, "", "")
	if err != nil {
		return nil, fmt.Errorf("failed to retrieve Kubernetes configuration and client for cluster with ID %s: %w", clusterID, err)
	}
```